Tools/ardupilotwaf/boards.py

```diff
@@ -223,6 +223,10 @@ def configure_env(self, cfg, env):
         ]
         env.LIB += cfg.env.LIB_RT
 
+        cfg.check_cfg(package='libsystemd', mandatory=False, global_define=True,
+                      args = ['--libs', '--cflags'])
```

Comment:
You can make a generic _check_library_ that just receives the package, but I think that having the checks together in the same place would be the best.